.circleci/config.yml

```diff
@@ -113,4 +159,15 @@ workflows:
             branches:
               only:
                 - staging
-                - master
\ No newline at end of file
+                - master
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - staging
+                - master
```

Comment:
I don't see why we wouldn't want these to run in staging as well, would like to keep the envs as close as possible and maybe we'll find out if something is broken in staging first.